application/console/config/main.php

```diff
@@ -2,23 +2,8 @@
 return [
     'id' => 'app-console',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'gii'],
-    'controllerNamespace' => 'console\controllers',
```

Comment:
if that's the case, I'll add it back it in.  For now, it's not needed on migrate or Gii calls so I removed it.